api/src/main/java/net/labymod/addons/waypoints/waypoint/Waypoint.java

```diff
@@ -1,24 +1,10 @@
 package net.labymod.addons.waypoints.waypoint;
 
-import net.labymod.api.client.component.Component;
 import net.labymod.api.client.world.object.WorldObject;
-import net.labymod.api.util.Color;
 
 public interface Waypoint extends WorldObject {
 
   WaypointMeta meta();
 
   WaypointObjectMeta waypointObjectMeta();
-
-  default Component title() {
-    return this.meta().getTitle();
-  }
-
-  default Color color() {
-    return this.meta().getColor();
-  }
-
-  default WaypointType type() {
-    return this.meta().getType();
-  }
```

Comment:
Please don't just remove/rename any methods in the api, for example the Minimap needs them.